### PR TITLE
Project setup (resolves #2)

### LINKED ISSUE #2: Create a "Hello World" backend
- [x] create initial controller
- [x] setup CORS

### PR DESCRIPTION
**Pull request by bambooSocks**
Changes introduced:
- added a test message controller
- setup a global CORS configurer to allow requests from all origins and for all mappings and headers, initially limited to only GET and POST requests
- added `package.json` with dependencies to run Prettier inside IntelliJ

### CODE REVIEW

File: src/main/java/org/autopotato/linkedinclonebackend/LinkedinCloneBackendApplication.java [hunk 1]
```diff
@@ -2,11 +2,33 @@
...
+     */
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
+        return new WebMvcConfigurer() {
+
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry
+                    .addMapping("/**")
+                    .allowedOrigins("*")
```
Comment (Krismix1) [inline src/main/java/org/autopotato/linkedinclonebackend/LinkedinCloneBackendApplication.java]: Let's limit the origins to only `http://localhost:4200`.

**Review by Krismix1 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
-linkedin-clone-backend
+# Linkedin Clone Backend
```
Comment (Krismix1) [inline README.md]: How do we want to capitalize the letters in the name?
In the frontend I was doing "LinkedIn Clone", but we just need to agree on something.
You choose

File: src/main/java/org/autopotato/linkedinclonebackend/LinkedinCloneBackendApplication.java (hunk 1 above)
Comment (bambooSocks) [reply src/main/java/org/autopotato/linkedinclonebackend/LinkedinCloneBackendApplication.java]: Ok I’ll change it as soon as I get home

File: README.md (hunk 2 above)
Comment (bambooSocks) [reply README.md]: I was going off of the logo but they seem to capitalize LinkedIn instead of Linkedin... so let’s go with that

**Review by Krismix1 — APPROVED** [review]

**Pull request merged**